blog-homepage/src/ForYouCard.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import AuthorBio from './AuthorBio';
+
+
+export default class ForYouCard extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isBookmarked: false,
+        }
+    }
+    
+
+    handleBookmark = (e) =>{
+        e.preventDefault();
+        
+        if(!this.state.isBookmarked){
```

Comment:
Nice! typically you'd just want this logic to be in your render function. Don't mess with traversing the document and adding classes, etc.

Reply:
Sorry I did not quite understand. If i include this inside render function, how should it work without the classes?

Reply:
I added a comment to your code below. Basically you want your render function to handle all the DOM manipulation.